docs/content/deployment/guides/kubernetes/deploying-with-helm-advanced.mdx

```diff
@@ -219,7 +219,7 @@ After Helm has successfully installed all the required kubernetes resources, sta
       -o jsonpath="{.items[0].metadata.name}")
     kubectl --namespace default port-forward $DAGIT_POD_NAME 8080:80
 
-Visit <http://127.0.0.1:8080>, and navigate to the [launchpad](http://127.0.0.1:8080/workspace/example_repo@k8s-example-user-code-1/jobs/celery_step_isolated_job/launchpad). Notice how `resources.io_manager.config.s3_bucket` is set to `dagster-test`. You can replace this string with any other accessible S3 bucket. Then, click _Launch Run_.
+Visit <http://127.0.0.1:8080>, and navigate to the [launchpad](http://127.0.0.1:8080/workspace/example_repo@k8s-example-user-code-1/jobs/pod_per_op_celery_job/launchpad). Notice how `resources.io_manager.config.s3_bucket` is set to `dagster-test`. You can replace this string with any other accessible S3 bucket. Then, click _Launch Run_.
```

Comment:
```suggestion
Visit <http://127.0.0.1:8080>, and navigate to the [launchpad](http://127.0.0.1:8080/workspace/example_repo@k8s-example-user-code-1/jobs/pod_per_op_celery_job/launchpad). Notice how `resources.io_manager.config.s3_bucket` is set to `dagster-test`. You can replace this string with any other accessible S3 bucket. Then, click **Launch run**.
```

Sorry if I missed this last time - please format UI elements (pages, buttons, links, etc) as bold